pkgs/development/libraries/abseil-cpp/default.nix

```diff
@@ -1,4 +1,11 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch, cmake, static ? stdenv.hostPlatform.isStatic }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, cmake
+, static ? stdenv.hostPlatform.isStatic
+, cxxStandard ? null
```

Comment:
```suggestion
, cxxStandard ? "17"
```

Reply:
I don't think this should be the default. One of the purposes of this change is to give `abseil-cpp` the default behavior of `-DCMAKE_CXX_STANDARD`, and not force transitive overrides unless necessary, which is so far only in the case of `or-tools`.